embulk-core/src/main/java/org/embulk/jruby/LazyScriptingContainerDelegate.java

```diff
@@ -104,7 +106,7 @@ private static LazyScriptingContainerDelegate of(
                             ? ScriptingContainerDelegate.LocalContextScope.SINGLETON
                             : ScriptingContainerDelegate.LocalContextScope.SINGLETHREAD,
                     ScriptingContainerDelegate.LocalVariableBehavior.PERSISTENT,
-                    initializer);
+                    initializer,logger);
```

Comment:
We have a space, or a newline, after a comma. In this case, preceding parameters are newline-separated, then let's follow them.

```suggestion
                    initializer,
                    logger);
```

Reply:
Thanks! done.